pytest/tests/sanity/routing_table_sync.py

```diff
@@ -6,12 +7,20 @@
 # edge pruning in order to eliminate those factors for sake of testing. In addition uses JsonRPC to add/remove edges
 # check current state of routing table on both sites.
 
-import sys, time
+import os
+import sys
+import time
 
 import base58
 import ed25519
 
-sys.path.append('lib')
+cur_dir = os.path.dirname(os.path.realpath(__file__))
+lib_dir = os.path.dirname(os.path.dirname(cur_dir))
+
+sys.path.append(os.path.join(lib_dir, "lib"))
```

Comment:
Like before, I’d go with `pathlib`, e.g.: `sys.path.append(pathlib.Path(__file__).resolve().parent.parent / 'lib')`.

The bigger problem with this is that if we want to do this we should do it in all the tests.